components/admin_console/workspace-optimization/dashboard.data.tsx

```diff
@@ -70,8 +71,18 @@ const useMetricsData = () => {
     };
 
     const getUpdatesData = (data: UpdatesParam) => ({
-        title: formatMessage({id: 'admin.reporting.workspace_optimization.updates.title', defaultMessage: 'Updates and Errors'}),
-        description: formatMessage({id: 'admin.reporting.workspace_optimization.updates.description', defaultMessage: 'You have an update to consider'}),
+        title: formatMessage({
+            id: 'admin.reporting.workspace_optimization.updates.title',
+            defaultMessage: 'Server Updates',
```

Comment:
Let's change "Updates" to "updates" if all other sections of this page are also sentence case.